src/asset_folder_verify_files.py

```diff
@@ -75,13 +77,13 @@
             logger.info("Found %d files existing, %d files missing\r" % (files_existing,files_nonexisting))
             c=0
 
-        filepath = os.path.join(fileref['filepath'],fileref['filename'])
+        filepath = os.path.join(fileref['filepath'],fileref['filename']).encode('utf8')
         if os.path.exists(filepath):
```

Comment:
does os.path.exists require a bytestring? why can't `filepath` be kept as a utf string throughout?

Reply:
Yes. Because it does not work under Python 3 if left as was.

Reply:
These changes may only be needed running in a Docker container under macOS and reading the file system from there. As they do not appear to break anything under Linux, I suggest we leave them.

If you like I could remove them and see if they are needed under Linux. This would be inconvenient, as I would have to keep putting them back in for testing in the Docker container.

Reply:
that's ok, but out of interest what is the issue? why is the filepath coming as bytes somethimes and utf other times? does it relate to the encoding of the filesystem?

Reply:
I do not know. I got various encoding errors with the old code. I tried various encoding commands to fix the errors. The commands you see in the code are the ones that worked.

I started the fixes on a Docker container running on macOS. Under Linux the changes may well still be needed, I have not tested this, so I can not say if the problems where related to the file system in macOS, but it is possible.